index.js

```diff
@@ -0,0 +1,55 @@
+require("dotenv").config();
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const products = require("./routes/products.router");
+const users = require("./routes/users.router");
+const carts = require("./routes/carts.router");
+const wishlists = require("./routes/wishlists.router");
+const initializeDbConnection = require("./db/db.connect");
+
+const app = express();
+
+app.use(bodyParser.json());
```

Comment:
Just to let you know, express now comes with json body parser by default, you can use that as well if you don't want to install extra dependencies. You can use it like this

    app.use(express.json())

Reply:
oh didn't knew this, thanks!. Will use this from now on